src/marqo/tensor_search/enums.py

```diff
@@ -5,12 +5,26 @@
 class MediaType:
     text = 'text'
     image = 'image'
+    video = 'video'
     default = 'text'
 
+class FileType:
+    youtube = "youtube"
+    tiktok = "tiktok"
+    url = "url"
+    local = "local_path"
+    straight_text = "straight_text"
+    ndarray = "ndarray"
+    PILImage = "PILImage"
+    ListOfPILImage = "ListOfPILImage"
```

Comment:
Can we keep these PIL image?
PILImage -> pil_image
ListOfPILImage -> list_of_pil_images
Unless there's a special reason for this casing

Reply:
done